cmd/iceberg/main.go

```diff
@@ -70,6 +71,7 @@ type Config struct {
 	Uuid     bool `docopt:"uuid"`
 	Location bool `docopt:"location"`
 	Props    bool `docopt:"properties"`
+	Create   bool `docopt:"create"`
```

Comment:
any way we could add a test of some kind for this? perhaps a dummy category type to confirm we call the function?

Reply:
@zeroshade I can create test to call `main` function with test arguments:
```
func TestCommands(t *testing.T) {
	os.Args = []string{"iceberg", "create", "namespace", "public"}
	main()
}
```
But I don't know how could I mock and verify call on `catalog`.
Otherwise it just tries to connect to real catalog and fails.

Reply:
fair enough. Something we should look into in the future being a way to mock the catalog stuff to create a framework for unit testing the CLI. Don't think we should prevent this getting merged waiting for that though.

Reply:
@zeroshade Resolved conflicts after pulling latest `main`, but actually there is not much to merge as similar changes in CLI were applied by #173 : just remove duplicated "create" command and unsupported error for "create table".